python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -302,6 +302,15 @@ interface {{ ethernet_interface.name }}
 {%     if ethernet_interface.l2_protocol.encapsulation_dot1q_vlan is arista.avd.defined %}
    l2-protocol encapsulation dot1q vlan {{ ethernet_interface.l2_protocol.encapsulation_dot1q_vlan }}
 {%     endif %}
+{%     if ethernet_interface.mac_timestamp is arista.avd.defined %}
+{%         if ethernet_interface.mac_timestamp == "header" %}
+   mac timestamp header
+{%         elif ethernet_interface.mac_timestamp == "before-fcs" %}
+   mac timestamp before-fcs
+{%         elif ethernet_interface.mac_timestamp == "replace-fcs" %}
+   mac timestamp replace-fcs
+{%         endif %}
+{%     endif %}
```

Comment:
As value is already validated by the schema - can we just render it right away without additional checks?
```suggestion
{%     if ethernet_interface.mac_timestamp is arista.avd.defined %}
   mac timestamp {{ ethernet_interface.mac_timestamp }}
{%     endif %}
```

Reply:
Fixed.